### PR TITLE
moved consumer, linting, add deps, removing extra stuff

### LINKED ISSUE #2: Move consumer.js from nodezoo-updater
Move to here.

### PR DESCRIPTION
**Pull request by Georgette**
closes #2

### CODE REVIEW

File: lib/dequeue.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+module.exports = function (options) {
+  var seneca = this
+
+  seneca.add('role:updater,info:update', onNpmChange)
+
+  function onNpmChange (pkg, respond) {
```
Comment (mcdonnelldean) [inline lib/dequeue.js]: Move function outside plugin, name of function should just be dequeue or infoUpdated.

File: lib/dequeue.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    var seneca = this
+    seneca.act({role: 'info', req: 'part', name: pkg.name}, (err) => {
+      respond(err, {message: `${pkg.name} package was updated `})
+    })
+  }
```
Comment (mcdonnelldean) [inline lib/dequeue.js:10]: At the end of each plugin you need to return the name for logging. Structure is,

```
return {
  name: 'nodezoo-dequeue'
}
```

File: package.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
   "description": "A message dequeuer for nodezoo",
   "scripts": {
     "start": "node srv/start.js",
-    "lint": "lab -P test -dL",
```
Comment (mcdonnelldean) [inline package.json:7]: Missing test script

File: package.json [hunk 4]
```diff
@@ -14,7 +14,8 @@
   ],
   "author": "Dean McDonnell (github.com/mcdonnelldean)",
```
Comment (mcdonnelldean) [inline package.json]: Author is GeGe not Dean

File: package.json [hunk 5]
```diff
@@ -14,7 +14,8 @@
...
   "contributors": [
-    "Shane Lacey (github.com/shanel262)"
+    "Shane Lacey (github.com/shanel262)",
+    "GeGe Pincin (github.com/Georgette)"
```
Comment (mcdonnelldean) [inline package.json]: GeGe as author, Not contributor

File: srv/start.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  }
+}
+
+var Service = Seneca()
+
+Service
+  .use(Entities)
+  .use(RedisQ, opts.redisQ)
+  .use('mesh', opts.mesh)
+  .use('../lib/dequeue.js')
```
Comment (mcdonnelldean) [inline srv/start.js]: Use require above (like entity) it stops weird bugs.

**Comment by mcdonnelldean** [comment]
Looks good, some notes for you.

**Comment by Georgette** [comment]
@mcdonnelldean Thanks for having my back, all done.

**Pull request merged**